iothub/device/src/IotHubClientOptions.cs

```diff
@@ -66,25 +67,24 @@ public IotHubClientOptions(IotHubClientTransportSettings transportSettings)
         /// <summary>
         /// The DTDL model Id associated with the device or module client instance.
         /// </summary>
-        /// <remarks>
-        /// This feature is currently supported only over MQTT and AMQP transports.
-        /// </remarks>
         public string ModelId { get; set; }
 
         /// <summary>
         /// The configuration for setting <see cref="Message.MessageId"/> for every message sent by the device or module client instance.
         /// </summary>
         /// <remarks>
-        /// The default behavior is that <see cref="Message.MessageId"/> is set only by the user.
+        /// The default behavior is that a message Id is sent only if set by the user.
         /// </remarks>
         public SdkAssignsMessageId SdkAssignsMessageId { get; set; } = SdkAssignsMessageId.Never;
 
         /// <summary>
         /// Sets the retry policy used in the operation retries.
         /// </summary>
         /// <remarks>
-        /// Defaults to a nearly infinite exponential backoff. If set to null, will use <see cref="IotHubClientNoRetry"/> to perform no retries.
-        /// Can be set to any of the built in retry policies such as <see cref="IotHubClientFixedDelayRetryPolicy"/> or <see cref="IotHubClientIncrementalDelayRetryPolicy"/> 
+        /// Defaults to a nearly infinite exponential backoff.
+        /// If set to null, will use <see cref="IotHubClientNoRetry"/> to perform no retries.
+        /// It can be set to any of the built-in retry policies such as <see cref="IotHubClientExponentialBackoffRetryPolicy"/>,
+        /// <see cref="IotHubClientFixedDelayRetryPolicy"/>, or <see cref="IotHubClientIncrementalDelayRetryPolicy"/>,
```

Comment:
I don't know about quickly going out of date, but just like `IAuthenticationMethod`, I think we need to tell someone what types are available to them to use, otherwise they have to hunt (not as easy when you aren't in azureiot.sln) or find a sample.